src/components/TopBar/TopBar.vue

```diff
@@ -303,17 +303,21 @@ export default {
 
 <style lang="scss" scoped>
 .top-bar {
+	--border-width: 1px;
+	// hardcoded 1.5 value of line-height for compatibility with older versions
+	--text-height: calc(1.5 * (var(--default-font-size) + var(--font-size-small, 15px)));
```

Comment:
Why default small size to 15px? Do we want to backport it to 29?

Reply:
Font size shouldn't change for Desktop client against 29, am I right?

Reply:
It will use server styles required for the current Talk version.

Otherwise it's more painful and we would need support not only different backend versions but also styles on frontend

Reply:
Please adjust in your follow up, when possible. I'm not getting it fully still 🤔

Reply:
Was 15px added here only for the Desktop client?

Reply:
Yes. Otherwise I don't see why we adjusted styles for stable versions before

Reply:
We adjusted in libraries like `@nextcloud/vue` because it is used on multiple versions.

Did we do it on Talk?

Reply:
https://github.com/nextcloud/spreed/commit/42e919117191e1bc266740991133a8166710fa2d

Reply:
This PR is not backported and adapts to new styles and variables, which is necessary for Talk web on 30 🤔

Reply:
I meant he iconSize, which depends on server styles and not the Talk version

https://github.com/nextcloud/spreed/commit/42e919117191e1bc266740991133a8166710fa2d#diff-841f2c9a1004d8fafe1e1f6fe44412b062860ea277a4ddc071068f7f2a162e7dR156-R157

Reply:
It's better to get it from variable anyway.